### DISCUSSION TITLE
[FE] 最優先ルーティング進捗

### DISCUSSION THREAD

**Discussion by kobakichi** [Hackathon5-TeamD/evolution-quest#56] [Q&A]
ルーティングの設定ですが、/でLogin表示、/registerでRegisterページを表示するところまで進みました。
Registerコンポーネント 現在はただ文字を表示するだけです。
まだpushしませんが、こんな感じで進めれば良いでしょうか？？？

**Comment by Terralien-jp** [comment]
全くもっていいと思います！ありがとうございます！

**Reply by kobakichi** [reply]
ありがとうございます。
このまま進めます！！
